modules/va_facilities/README.v1.yml

```diff
@@ -445,9 +425,13 @@ paths:
         services. Only facilities of type "health" and "benefits" may be filtered by available
         services.
 
-        Results of this operation are paginated. JSON responses include pagination information
-        in the standard JSON API "links" and "meta" elements. GeoJSON responses include pagination
-        information in the "Link" header.
+        Results of this operation are paginated. Responses include pagination information
+        in the standard JSON API "links" and "meta" elements.
+
+        Also in the "meta" element is information about the drive time band that the requested location 
+        fell within for each the facility included in the response. The values of drive_time_band_min and 
+        drive_time_band_max are in minutes. For example, a facility returned with a matched drive_time_band_min 
+        of 10 and drive_time_band_max of 20 is a 10 to 20 minute drive from the requested location.
 
         The facility information returned by `/nearby` is a small subset of the information returned by `/facilities`.
```

Comment:
I wouldn't even call the id link a "small subset of the information." Would it be better to say something like: "This endpoint does not return facility information. To retrieve details for facilities found by `/nearby` follow the relationship links and check documentation for `/facilities`."